.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
+# DOCKER_USER
+# DOCKER_PASS
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:18.03.0-ce
+    working_directory: /dockerflow
+    steps:
+      - run:
+          name: Install Docker build dependencies
+          command: apk add --no-cache openssh-client git bash
+      - checkout
+      - setup_remote_docker
+
+      - run:
+          name: Create a version.json
+          command: |
+            # create a version.json per https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md
+            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
+            "$CIRCLE_SHA1" \
+            "$CIRCLE_TAG" \
+            "$CIRCLE_PROJECT_USERNAME" \
+            "$CIRCLE_PROJECT_REPONAME" \
+            "$CIRCLE_BUILD_URL" > version.json
+
+      - run:
+          name: Build Docker image
+          command: docker build -f Dockerfile -t fxa-email-service:build .
```

Comment:
Gonna open a follow up issue to somehow `cargo test` the thing before it ships, but for now we are covered by Travis